Rocket.Chat/Controllers/Settings/SettingsViewController.swift

```diff
@@ -34,6 +34,9 @@ final class SettingsViewController: UITableViewController {
 
     func cellContactDidPressed() {
         if !MFMailComposeViewController.canSendMail() {
+            let alert = UIAlertController(title: "No Mail Accounts", message: "Please set up a Mail account in order to send email.", preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: localized("global.ok"), style: .default, handler: nil))
```

Comment:
Let's use translated strings here? You can translate to English only if you want. We've a method called `localized()` for that.